sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/CallConnectionStateChangedEvent.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+import com.azure.communication.callingserver.implementation.models.CallConnectionStateChangedEventInternal;
+import com.azure.communication.callingserver.models.CallConnectionState;
+import com.azure.core.util.BinaryData;
+
+/** The call connection state changed event. */
+public final class CallConnectionStateChangedEvent {
```

Comment:
Optionally, annotate this with "Immutable"

Reply:
all immutable objects are marked as '@Immutable'.